packages/react/src/components/ActionList/ActionListLinkItem.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { forwardRef, useMemo } from 'react';
+import classnames from 'classnames';
+import ActionListItem from './ActionListItem';
+import { useActionListContext } from './ActionListContext';
+
+type ActionListLinkItemProps = React.AnchorHTMLAttributes<HTMLAnchorElement> &
+  Omit<
+    React.ComponentProps<typeof ActionListItem>,
+    keyof React.HTMLAttributes<HTMLLIElement> | 'selected'
+  > & { href: string };
+
+const ActionListLinkItem = forwardRef<
+  HTMLAnchorElement,
+  ActionListLinkItemProps
+>(
+  (
+    {
+      key,
+      className,
+      // ActionListLinkItem should not be able to be "selected"
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      selected,
+      ...props
+    },
+    ref
+  ) => {
+    const { role: contextRole } = useActionListContext();
+    const listItemRole = useMemo(() => {
+      if (contextRole === 'menu') {
+        return 'menuitem';
+      }
+
+      // istanbul ignore next
+      if (contextRole && process.env.NODE_ENV !== 'production') {
+        console.warn(
```

Comment:
If we know this will cause accessibility issues do we want to throw an error instead?

Reply:
No. It _may_ cause an an accessibility issue but we don't know the surrounding context. We shouldn't prevent the app from rendering because there might or might not be a problem.